packages/jest-runner/src/runTest.ts

```diff
@@ -104,16 +104,19 @@ async function runTestInternal(
     });
   }
 
+  const Runtime: typeof RuntimeClass = config.moduleLoader
+    ? require(config.moduleLoader)
+    : require('jest-runtime');
+  const transformer = new Runtime.ScriptTransformer(config);
+
```

Comment:
Instead of going via `jest-runtime`, could you import directly from `@jest/transform`?

Reply:
What is the significance of L108? 

Runtime is created from either `config.moduleLoader` or `jest-runtime`.
I thought there was some significance related to the which was chosen.

Reply:
It makes the runtime module pluggable. The script transformer was extracted from the runtime (#7915), but kept the export for backwards compat. For our purposes here, importing directly from `@jest/transform` makes more sense

Reply:
I'll use it from `@jest/transform` then